frontend/packages/kubevirt-plugin/src/k8s/requests/config-map/storage-class/storageClassConfigMap.ts

```diff
@@ -0,0 +1,44 @@
+import { ConfigMapModel } from '@console/internal/models';
```

Comment:
It's a repeating story - I would not copy&paste the code from web-ui-components here without removing it there.

I would migrate web-ui-components to openshift/console at once - ideally in 4.3 if we get time for it.
Otherwise we will need to maintain the almost same code twice - with just tiny, hard to spot, differences on both sides. which will cause extra effort and regressions.

Reply:
Moving it whole cannot be done easily - it should be done by parts and there will always be duplicate code until everything is ported (because of the not so straightforward dependencies)

removed for now on, but still this will appear in the next patch for 4.3